src/components/modal/modal.css

```diff
@@ -0,0 +1,40 @@
+.modal-wrapper {
+  z-index: 2;
+  position: fixed;
+  top: 0;
+  width: 100vw;
+  height: 100vh;
+  display: none;
+  justify-content: center;
+  align-items: center;
+  backdrop-filter: blur(1px);
+}
+
+.modal {
+  border-radius: 8px;
+  border: 1px solid var(--BLACK);
+  background-color: var(--WHITE);
+  max-width: 350px;
+  padding: 10px;
```

Comment:
Why have you used pixels over here?

Reply:
i don't want the width of modal to be greater than the mobile screen width